### PR TITLE
Many to many

### PR DESCRIPTION
**Pull request by rufinogs**

### CODE REVIEW

**Comment by rufinogs** [comment]
Error:  COMPILATION ERROR : 
[1454](https://github.com/URJC-CP/iudex/runs/6166207176?check_suite_focus=true#step:8:1454)
[INFO] -------------------------------------------------------------
[1455](https://github.com/URJC-CP/iudex/runs/6166207176?check_suite_focus=true#step:8:1455)
Error:  /home/runner/work/iudex/iudex/src/main/java/es/urjc/etsii/grafo/iudex/services/SubmissionService.java:[131,49] incompatible types: es.urjc.etsii.grafo.iudex.entities.Contest cannot be converted to es.urjc.etsii.grafo.iudex.entities.ContestTeams
[1456](https://github.com/URJC-CP/iudex/runs/6166207176?check_suite_focus=true#step:8:1456)
Error:  /home/runner/work/iudex/iudex/src/main/java/es/urjc/etsii/grafo/iudex/services/SubmissionService.java:[132,50] incompatible types: es.urjc.etsii.grafo.iudex.entities.Problem cannot be converted to es.urjc.etsii.grafo.iudex.entities.TeamsProblems
[1457](https://github.com/URJC-CP/iudex/runs/6166207176?check_suite_focus=true#step:8:1457)
Error:  /home/runner/work/iudex/iudex/src/main/java/es/urjc/etsii/grafo/iudex/services/ContestService.java:[148,36] cannot find symbol
[1458](https://github.com/URJC-CP/iudex/runs/6166207176?check_suite_focus=true#step:8:1458)
  symbol:   method getListaParticipantes()
[1459](https://github.com/URJC-CP/iudex/runs/6166207176?check_suite_focus=true#step:8:1459)
  location: variable contest of type es.urjc.etsii.grafo.iudex.entities.Contest
[1460](https://github.com/URJC-CP/iudex/runs/6166207176?check_suite_focus=true#step:8:1460)
Error:  /home/runner/work/iudex/iudex/src/main/java/es/urjc/etsii/grafo/iudex/services/ContestService.java:[180,28] incompatible types: es.urjc.etsii.grafo.iudex.entities.Problem cannot be converted to es.urjc.etsii.grafo.iudex.entities.ContestProblem
[1461](https://github.com/URJC-CP/iudex/runs/6166207176?check_suite_focus=true#step:8:1461)
Error:  /home/runner/work/iudex/iudex/src/main/java/es/urjc/etsii/grafo/iudex/services/ContestService.java:[209,31] incompatible types: es.urjc.etsii.grafo.iudex.entities.Problem cannot be converted to es.urjc.etsii.grafo.iudex.entities.ContestProblem
[1462](https://github.com/URJC-CP/iudex/runs/6166207176?check_suite_focus=true#step:8:1462)
Error:  /home/runner/work/iudex/iudex/src/main/java/es/urjc/etsii/grafo/iudex/services/ContestService.java:[237,21] cannot find symbol

**Comment by rufinogs** [comment]
este error no se como solucionarlo 'addProblem(es.urjc.etsii.grafo.iudex.entities.ContestProblem)' in 'es.urjc.etsii.grafo.iudex.entities.Contest' cannot be applied to '(es.urjc.etsii.grafo.iudex.entities.Problem)' en Problem Service.java en la linea 117

**Review by rmartinsanta — CHANGES_REQUESTED** [review]
Esta pendiente finalizar la rama, vuelve a pedir al review cuando esté lista para revisión.

File: src/main/java/es/urjc/etsii/grafo/iudex/services/OnStartRunner.java [hunk 1]
```diff
@@ -7,13 +7,15 @@
...
 import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Service;
 
 import java.io.File;
 import java.time.LocalDateTime;
 import java.util.Optional;
 import java.util.TimeZone;
 
 @Component
+@Service
```
Comment (rmartinsanta) [inline src/main/java/es/urjc/etsii/grafo/iudex/services/OnStartRunner.java:18]: ¿Por qué es component y service? ¿No sería suficiente con una de ellas?

**Review by rmartinsanta — APPROVED** [review]
Buen trabajo, revisa el comentario y los conflictos y mergea

**Pull request merged**